serverless/openvino/omz/intel/face-detection-0205/function.yaml

```diff
@@ -0,0 +1,61 @@
+metadata:
+  name: openvino-omz-face-detection-0205
+  namespace: cvat
+  annotations:
+    name: Attributed face detection
+    type: detector
+    framework: openvino
+    # attribute names have to be the same as in annotated task, otherwise values will be ignored
+    spec: |
+      [
+        { "id": 0, "name": "face", "attributes": ["age", "gender", "emotion"]}
```

Comment:
@mikhail-treskin , thanks for the great PR. Could you please help us to polish the functionality? Each attribute should have a type and possible values. For example, when CVAT returns an attribute (e.g. GET /api/v1/tasks), it returns something like the structure below:

```
            "labels": [
                {
                    "id": 0,
                    "name": "age",
                    "attributes": [
                        {
                            "id": 0,
                            "name": "age",
                            "input_type": "number",
                            "default_value": "25",
                            "values": [
                                "0",
                                "150",
                                "1"
                            ]
                        }
                    ]
                }
            ],
```

Do you think it is valuable to have something like that here? Otherwise it is unclear which values each attribute has. Also it will be difficult to much them to a corresponding task. Basically, I should be able to take labels and attributes from the definition as is and create a task with them.

Reply:
Yes, I agree that possible values should be part of function config. 
But what about attribute type, i'm not really sure that it does make sense to store it in function config since attribute types is CVAT specific information. Maybe it make sense to add additional attributes verification on backend side, e.g. if certain attribute has type "Select" or "Radio" only one value can be accepted, and check if "Number" attribute coming from nuclio  is really convertible to number. Otherwise exception can be raised or fallback to default attribute value. 
What do you think about such an approach?

Reply:
@mikhail-treskin , `spec` section inside function.yaml is also CVAT specific. Basically I believe that CVAT labels and attributes definition and serverless labels and attributes definition should correspond to each other. Also in the future it is good to implement matching of attributes by name in UI (as it is implemented for labels). It will be necessary to understand the type of an attribute and possible values. I don't suggest implementing matching in UI for the PR. It is another story.

Proposal:
- Let's have the same definition for attributes as CVAT server support (see an example above). 
- If an attribute is supported by a serverless function, but it isn't supported by a corresponding label in CVAT, it should be ignored.
- If attributes are matched by name, it is necessary to see if we can cast it to a corresponding type. For example, any attribute can be converted to `text` type. At the same time `text` type cannot be converted to `select`. An attribute, which is matched by name but cannot be casted, should be ignored.

Thus we have a set of labels with attributes from a task. Also we have a number of labels with attributes from a serverless function. We match labels by name. After that we match attributes by name. After that we match attributes by type. During the process some labels and attributes will be filtered. The final list of labels with reduced number of attributes can be covered by our serverless function.
It is a typical case when a serverless function supports more labels and attributes when it is necessary. All of them should be ignored as it is implemented for labels.

Reply:
@nmanovic Could you please have a look on logic of attributes filtering in lambda manager in latest [commit](https://github.com/openvinotoolkit/cvat/pull/3917/commits/0774a4cdaa6194a26563591edb4f80b8efad1a62)?

If it looks fine for you I'll implement same logic in FE in cvat-ui/src/components/annotation-page/standard-workspace/controls-side-bar/tools-control.tsx